fastapi_opa/auth/auth_saml.py

```diff
@@ -29,15 +40,65 @@ async def authenticate(
         auth = await self.init_saml_auth(request_args)
 
         if "acs" in request.query_params:
+            print(datetime.utcnow(), '--acs--')
```

Comment:
Please do not print anything but use the logger instead because it can be handled much more flexible by the library user.

Reply:
Changed